lib/i18n/onboarding.i18n.dart

```diff
@@ -1,29 +1,25 @@
-
-
 import 'package:i18n_extension/i18n_extension.dart';
 
 extension Localization on String {
+  static final _t = Translations.byLocale('en_us') +
+      {
+        'es_es': {
+          // TRANSLATION NOTE: These are automatic translations by Google Translate - please replace
 
-static final _t = Translations.byLocale('en_us') +
-    {
-      'es_es': {
-        
-        // TRANSLATION NOTE: These are automatic translations by Google Translate - please replace
-
-        'Better than free transactions': 'Transacciones mejor que gratis',
-        "Make payments globally without any fees. Earn rewards when you support 'regenerative' organisations and people.":
-        "Realiza pagos a nivel mundial sin ningún cargo. Obtenga recompensas cuando apoye a organizaciones y personas 'regenerativas'.",
-        'Vote directly on social and environmental impact projects you care about.':
-        'Vota directamente sobre los proyectos de impacto social y ambiental que te interesan.',
-        'Citizen Campaigns': 'Campañas ciudadanas',
-        'Unite with a global movement of organisations and people to regenerate our planet and heal our economy.': 
-        'Únete a un movimiento global de organizaciones y personas para regenerar nuestro planeta y sanar nuestra economía.',
-        'Regenerative Economy': 'Economía regenerativa',
-        'NEXT': 'SIGUIENTE',
-        'BACK': 'ATRÁS',
-        'JOIN NOW': 'ÚNETE AHORA',
-      }
-    };
+          'Better \nThan Free Transactions': 'Transacciones mejor que gratis',
```

Comment:
i see a `\n` new line char added here, is this on purpose?

Reply:
Good question! 🤓 This was updated to match the widget title in [FirstPage](https://github.com/JoinSEEDS/seeds_light_wallet/blob/master/lib/v2/screens/onboarding/components/pages/onboarding_page_1.dart#L11), which I suspect was done to break up the line for aesthetics? 🧐 Removing `\n` makes sense to me, but perhaps there's history behind this decision? @gguij004? 🤓

Reply:
![image](https://user-images.githubusercontent.com/8041901/128022610-184b52de-b77b-4a5d-a00f-dea9dd408142.png)

Reply:
i guess it was to match designs

Reply:
Hi (: .Yes, I was trying to match design on figma.